managed/src/main/java/com/yugabyte/yw/controllers/CloudProviderController.java

```diff
@@ -42,6 +42,7 @@
    *
    * @return JSON response of newly created provider
    */
+  @ApiOperation(value = "createCloudProvider", nickname = "createCloudProvider")
```

Comment:
I think someday I can change this to generate operationId automatically. For now lets live with this.